signature-specification.md

```diff
@@ -77,19 +77,19 @@ Notary v2 supports [JWS JSON Serialization](https://datatracker.ietf.org/doc/htm
 Notary v2 requires Payload to be the content **descriptor** of the subject manifest that is being signed.
 
 1. Descriptor MUST contain `mediaType`, `digest`, `size` fields.
-1. Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules](https://github.com/opencontainers/image-spec/blob/main/annotations.md#rules).
-   In Notary v2 annotations are being used to store signed attributes.
-   The annotations key prefix for Notary v2 use is not yet finalized.
-   See [issues-106](https://github.com/notaryproject/notaryproject/issues/106).
+1. Descriptor MAY contain `annotations` and if present it MUST follow the [annotation rules](https://github.com/opencontainers/image-spec/blob/main/annotations.md#rules). Notary v2 uses annotations for storing both Notary specific and user defined signed attributes. The prefix `org.cncf.notary` in annotation keys is reserved for use in Notary v2 and MUST NOT be used outside this specification.
```

Comment:
annotation namespaces is pending, per https://github.com/notaryproject/notaryproject/issues/106
Suggest we leave this for consistency in this PR and resolve as we finalize 106.